### PR TITLE
ci: Update slack reporting for XTS failure and CITR build promotion

### LINKED ISSUE #16731: CITR Phase 1: Update Slack Reporting
- Update slack messaging to report build promotion
- Move XTS Failures into the [#continuous-integration-test-failures](https://swirldslabs.slack.com/archives/C07PMU4MWDT) channel

### PR DESCRIPTION
**Pull request by rbarkerSL**
**Description**:

Update slack reporting to change the webhook secret name for messages targeting the `continuous-integration-test-failures` channel

Added slack reporting for build candidate promotion reporting targeting the `continuous-integration-test-operations` channel

*Note: the SLACK_CITR_WEBHOOK secret can be removed on merge of this PR*

**Related Issue(s)**:

Fixes #16731

### CODE REVIEW

File: .github/workflows/zxcron-promote-build-candidate.yaml [hunk 1]
```diff
@@ -145,6 +149,74 @@ jobs:
...
+        uses: slackapi/slack-github-action@6c661ce58804a1a20f6dc5fbee7f0381b469e001 # v1.25.0
+        env:
+          SLACK_WEBHOOK_TYPE: INCOMING_WEBHOOK
+          SLACK_WEBHOOK_URL: ${{ secrets.SLACK_CITR_OPERATIONS_WEBHOOK }}
+        with:
+          payload: |
+            {
+              "attachments": [
+                {
+                  "color": "#7647cd",
```
Comment (andrewb1269hg) [inline .github/workflows/zxcron-promote-build-candidate.yaml:177]: Is the color purple the right color to use here? When I see purple I think "merge complete" but wanted to see if white or gray made more sense

Comment (nathanklick) [reply .github/workflows/zxcron-promote-build-candidate.yaml:177]: If its a failure, then I would expect to see red or yellow. If its a passing report, I would like to see green or blue. If it is an informational report, then grey or white would be appropriate.

Comment (rbarkerSL) [reply .github/workflows/zxcron-promote-build-candidate.yaml]: I moved the failures to red and the success/promotion to green.

**Review by andrewb1269hg — APPROVED** [review]
Updated colors look great!

**Pull request merged**